junit-platform-commons/src/main/java/org/junit/platform/commons/support/ModifierSupport.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.commons.support;
+
+import static org.apiguardian.api.API.Status.MAINTAINED;
+
+import java.lang.reflect.Member;
+
+import org.apiguardian.api.API;
+import org.junit.platform.commons.util.Preconditions;
+import org.junit.platform.commons.util.ReflectionUtils;
+
+/**
+ * Common modifier support.
```

Comment:
Please expound a bit more here -- for example, something similar to what you wrote in the User Guide and Release Notes.

Reply:
I've adopted this comment from the other support classes. The JavaDoc for those should be updated as well. Will do that in one of the next PRs.

Reply:
OK. Thanks for submitting the other PR.